testCases/small.ledger

```diff
@@ -0,0 +1,5 @@
+2013/4/1
+  Expenses                   400
+  Assets
+
```

Comment:
This test case is very small. Could you add more journal entries to it? Or, alternatively, create a different test case with more entries? Thanks!